changelog.d/13652.feature

```diff
@@ -0,0 +1 @@
+Improve aesthetics of HTML templates.
```

Comment:
This seems to have pretty big backwards compatibility ramifications (if someone customized their templates). I think we at least need to leave some upgrade notes?

Reply:
I believe that way I've done this now isn't backwards incompatible (if someone has custom templates they'll still continue working fine).